python/lsst/ts/observatory/control/mock/mtcs_async_mock.py

```diff
@@ -153,6 +153,7 @@ async def setup_types(self) -> None:
 
         # MTM2 data
         self._mtm2_evt_force_balance_system_status = types.SimpleNamespace(status=False)
+        self._mtm2_evt_hardpointList = types.SimpleNamespace(actuators=[6])
```

Comment:
I think you mean 

```
self._mtm2_evt_hardpointList = types.SimpleNamespace(actuators=list(range(6)))
```

Reply:
why the `range(6)`? 
I could copy the list they give here:
https://github.com/lsst-ts/ts_m2/blob/develop/tests/test_csc.py#L157-L160

Reply:
right.. that is probably a better idea.. I naively thought it was from 0-5....